testnets/nibirudevnet3evm/assetlist.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "nibirudevnet3evm",
```

Comment:
This naming doesn't follow current usage at the Chain Registry. Can we use 'nibirudevnet4' instead (it doesn't matter that it doesn't match the chain_id)?